### ISSUE TITLE
My issue

### ISSUE THREAD

**Issue by tannerholcomb222** [MaxIsHot/kinektic#4]
Every time I place my digital miner it disappears.  Usually when I have been logged off for more than an hour and get back on.

**Comment by ghost** [comment]
Yes we are aware of this bug but were unsure of how to fix it. Mostly we think these Digital Miners are being lost because they were possibly placed in a unloaded chunk. We have banned chunk loaders for lagging the server but ill do some more research on how to fix this bug. If you want to read up on the issue you can here: https://github.com/aidancbrady/Mekanism/issues/4226
